app/domain/authentication/authn_jwt/jwt_fetch_jwks_uri_signing_key.rb

```diff
@@ -0,0 +1,51 @@
+require 'uri'
+require 'net/http'
+
+module Authentication
+  module AuthnJwt
+
+    JwtFetchJwksUriSigningKey = CommandClass.new(
+      dependencies: {
+        logger: Rails.logger,
+        discover_identity_provider: OAuth::DiscoverIdentityProvider.new,
+        fetch_secrets: Conjur::FetchRequiredSecrets.new
```

Comment:
it is best to match the param name with the class it uses so please change to `fetch_required_secrets`